html/php-pre-challenge3/index.php

```diff
@@ -1,6 +1,58 @@
 <?php
 $limit = $_GET['target'];
-
+$limit = mb_convert_kana($limit, 'a' , 'UTF-8');
+if(!is_numeric($limit) || $limit <= 0 || preg_match('/^([1-9]\d*|0)\.(\d+)?$/', $limit)) {
+    http_response_code(400);
+}
```

Comment:
ガードを突破したらlimitを整数型にキャストしましょう。後々厳密な比較 '===' を使いたいため

```suggestion
}
$limit = intval($limit);
```